### ISSUE TITLE
Contrast ratio for text that is part of inactive UI component

### ISSUE THREAD

**Issue by iamrafan** [w3c/wcag#469]
![t](https://user-images.githubusercontent.com/22068434/45124150-17216580-b11e-11e8-9a67-5f78d8a6ec24.png)

Shouldn't the text ```Link``` & ```Action``` have a min contrast ratio of 4.5:1 as the users with low vision need to have the same experience as the users without any visual impairment?

**Comment by awkawk** [comment]
Thank you for the comment - the WCAG 2.1 specification does not require a minimum contrast ratio for inactive elements ("Incidental: Text or images of text that are part of an inactive user interface component"). Having a higher contrast level for text of this sort can be regarded as a best practice even though it isn't required. This is a gap that may be able to be addressed in future versions through personalization.

**Comment by alastc** [comment]
Noting that we revisited this issue during the non-text contrast work in [WCAG21 issue 805](https://github.com/w3c/wcag21/issues/805). It was not possible to find a universally acceptable way of showing inactive/disabled element without using a lower-contrast for those elements.

**Comment by Gumnos** [comment]
In [a recent discussion about contrast](https://twitter.com/marcysutton/status/1313906290700161024), I had asked if there was any sort of markup/CSS to convey that certain text is "[incidental text](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)" and thus should be ignored for purposes of contrast testing. Neither of us knew of any such existing hooks, but @MarcySutton suggested I raise the idea on this issue. I'd like a way to prevent Chromium's #a11y tools from complaining that "Background and foreground colors do not have a sufficient contrast ratio" for particular incidental elements so its reports only list actual concerns.

I'm not sure whether it would be better to do this in the HTML DOM or in the CSS, but it seems functionally similar to the `user-select` CSS property.

If this would be better suited for a new issue rather than riding on the coat-tails of this issue, I'd be glad to do so.

**Comment by patrickhlauke** [comment]
i'd suggest opening a new issue, as there's no point piggy-backing onto an issue that was closed over 2 years ago...
